ShopperInsights/build.gradle

```diff
@@ -0,0 +1,61 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
```

Comment:
Let's add the dokka plugin here too so we can have kotlin docs for these new classes